test/known_issues/test-inspector-cluster-port-clash.js

```diff
@@ -10,9 +11,13 @@ const common = require('../common');
 //
 // Refs: https://github.com/nodejs/node/issues/13343
 
-common.skipIfInspectorDisabled();
+if (process.config.variables.v8_enable_inspector === 0) {
+  // When the V8 inspector is disabled, using either --without-inspector or
+  // --without-ssl, this test will not fail which it is expected to do.
+  // The following fail will allow this test to be skipped by failing it.
```

Comment:
Could you add something to this comment about replacing this block with `common.skipIfInspectorDisabled()` when/if this test is moved out of `known_issues`.